timesketch/frontend-ng/src/components/Visualization/AggregationConfig.vue

```diff
@@ -0,0 +1,366 @@
+<!--
+Copyright 2024 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<template>
+  <v-container class="ma-0">
+    <v-row>
+      <v-col cols="12" md="12">
+        <TsEventFieldSelect
+          :field="selectedField"
+          @selectedField="selectedField = $event"
+          :rules="[rules.required]"
+        >
+        </TsEventFieldSelect>
+      </v-col>
+    </v-row>
+    <v-row>
+      <v-col>
+        <v-autocomplete
+          outlined
+          v-model="selectedAggregator"
+          :disabled="disableAggregator"
+          :items="aggregators"
+          label="Aggregate events by"
+          :rules="[rules.required]"
+        ></v-autocomplete>
+      </v-col>
+    </v-row>
+    <v-row v-if="!disableMetric">
+      <v-col>
+        <v-autocomplete
+          outlined
+          v-model="selectedMetric"
+          :items="metrics"
+          label="Aggregation Metric"
+        ></v-autocomplete>
+      </v-col>
+    </v-row>
+    <v-row v-if="!disableMaxItems">
+      <v-col>
+        <v-text-field
+          :label="labelMaxItems"
+          outlined
+          v-model="selectedMaxItems"
+          single-line
+          type="number"
+          min=1
+        />
```

Comment:
I think adding the `.number` helped resolve the console error.  As an added side-effect it may have fixed the issue here https://github.com/google/timesketch/pull/3040#discussion_r1609933790

Reply:
Marking this as done.  Please re-open if its still causing issues